prj.conf

```diff
@@ -124,4 +124,7 @@ CONFIG_XRUN_SHELL_CMDS=y
 
 # Enable vchan
 
+CONFIG_CACHE=y
 CONFIG_XEN_VCH=y
+CONFIG_ARCH_CACHE=y
```

Comment:
This is not vchan config. Move at the top to system params. Also add comment that it is enable due to problem with vchan and TODO need to fix.